docs/user-guide/settings/cloud-sync.md

```
@@ -152,14 +155,20 @@ POST {server_url}/api/v1/sync/push
 
 ## 7. 同步历史
 
-### 7.1 表格列
+###7.1表格列
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)